frontend/packages/dev-console/src/components/pipelines/pipeline-builder/task-sidebar/TaskSidebarParam.tsx

```diff
@@ -13,7 +13,7 @@ const TaskSidebarParam: React.FC<TaskSidebarParamProps> = (props) => {
   const { hasParamError, onChange, resourceParam, taskParam } = props;
   const [dirty, setDirty] = React.useState(false);
 
-  const currentValue = taskParam.value?.toString() || '';
+  const currentValue = taskParam?.value?.toString() || '';
```

Comment:
Yeah, this is what I initially saw... I am curious to know if it somehow broke earlier in the flow. This can be a work around though.

Reply:
Ah, okay, I see how this happened. 

When the Pipeline Builder is creating the tasks, it helps out the user by giving the [needed defaults  as values](https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/pipelines/pipeline-builder/utils.ts#L33-L36) so if a minor change is needed, they have the value to do so (instead of a placeholder). However, when you're editing a Pipeline it does not do the same, it [just copies what it needs over](https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/pipelines/pipeline-builder/utils.ts#L108).

I don't see a super easy solution today for this. I'll need to try and clean this up when we top-level the cluster and namespace tasks for proper validation.

Ah well, this will have to do for 4.4.